### ISSUE TITLE
Demo input css

### ISSUE THREAD

**Issue by thomas-tran** [zachsoft/Ionic-Material#74]
How can I change the background color of the ion-md-input , it seems that ionic material set the background to black by default and I am unable to change it even I copy the stylesheet from ion-md-input itself.

**Comment by mhartington** [comment]
ion-md-input author here, you should be able to pass a highlight color to the directive. 

https://github.com/mhartington/ion-md-input#usage

```
<ion-md-input placeholder="Name" hightlight-color="energized" type="text"></ion-md-input>
```

**Comment by thomas-tran** [comment]
Hi 

My default color is white on the background page therefore I need change to style to white background color of the input box and the floating label to dark color. 

What I mean is the background color of the input text, at the moment it set to dark #444 in your style.css and I could not change the style unless overwrite your css.

The hightlight-color is just change the underline color of the input text and it did not change the background of the input.

**Comment by sangregoriopaolo** [comment]
Same issue for me

**Comment by mhartington** [comment]
So, I'm not sure I follow. Are you talking about the background color of the inputs or the text-color of the label? Either of the two can be overwritten with your own css, so I'm not sure this is much of an issue.

http://codepen.io/mhartington/pen/zGNrzV

It just would require you to write your own css to fit your needs.

**Comment by sangregoriopaolo** [comment]
Hi, actually, including your css, and following the instructions on ion-md-input to use it, I get this
![screen shot 2015-05-28 at 22 32 27](https://cloud.githubusercontent.com/assets/169325/7870418/8359dbea-0589-11e5-9fb0-b128ae51ac8f.png)

**Comment by mhartington** [comment]
Any way you could put together a codepen demo? 
Something else must be happening here that I'm not aware of

**Comment by thomas-tran** [comment]
Hi ,

I means the background of the input, how can you change it to dark color to white color? Also, the input floating label as well.

I used bower to install to my application and reference the css and js, however I could not find any document how to change the style of those input.

Thanks
Thomas

**Comment by raibutera** [comment]
Codepen or Plunkr please

**Comment by raibutera** [comment]
Please demonstrate this issue with a [codepen](http://ionicmaterial.readme.io/v0.3/docs/how-to-make-a-codepen)

**Comment by sliontc** [comment]
It likes the ionic-material conflict with ion-md-input.

**Comment by sliontc** [comment]
ionic-material overrides the ion-md-input.

**Comment by sliontc** [comment]
Just include both ionic-material and ion-md-input to reproduce the issue.

**Comment by sliontc** [comment]
Any feedback？

**Comment by liuwenzhuang** [comment]
Yes, if use ionic-material and ion-md-input together, the background of ion-md-input will change.

**Comment by Vaelyr** [comment]
.item-md-label input classes conflict. Need custom override to fix.

**Comment by adskiremote** [comment]
Hi - I've added a new directive to the ion-md-input.  This now resets the style.  If you want to use without Ionic Material styling just use like this: <ion-md-input-cus></ion-md-input-cus>

My fork here: https://github.com/adskiremote/ion-md-input

**Comment by mhartington** [comment]
Pretty sure this was fixed in my original repo, just this repo was pulling an older version

**Comment by adskiremote** [comment]
Hi Mike,
ah sorry didn't realise.  Thought the other was the updated.  Thanks for that as I'll update mine and check :-)

**Comment by megabitplanet** [comment]
1 add via bower ionic-material and ion-md-input.
2 in bower component ion-md-input open bower.json and modify main like that "main": [
        "ion-md-input.js",
        "css/ion-md-input.css"
    ],
3 open ionic-material.css in bower components and comment all row in section /* Directives : MD Label.
